client/service/client.go

```diff
@@ -59,6 +59,11 @@ const (
 
 var clientAuthorizerMutex sync.Mutex
 
+// DBApiClientMetadata used for unit testing and verifying client behavior
+type DBApiClientMetadata struct {
+	AzureAuthPatTokenExpiry int32
```

Comment:
why do we heed a struct just with single field? looks like anti-pattern

Reply:
noted and removed 😄